package.json

```diff
@@ -64,22 +70,16 @@
     "grunt-terser": "^0.1.0",
     "grunt-text-replace": "~0.4.0",
     "gruntify-eslint": "^4.0.0",
+    "hash.js": "^1.1.3",
     "istanbul": "^0.4.5",
     "mocha": "^5.0.0",
-    "sinon": "^4.3.0",
-    "text-encoding-utf-8": "^1.0.2",
-    "whatwg-fetch": "^2.0.3",
-    "@mattiasbuelens/web-streams-polyfill": "^0.3.1",
-    "asmcrypto.js": "github:openpgpjs/asmcrypto#6e4e407b9b8ae317925a9e677cc7b4de3e447e83",
-    "bn.js": "^4.11.8",
-    "buffer": "^5.0.8",
-    "elliptic": "github:openpgpjs/elliptic#6b7801573b8940a49e7b8253176ece2725841efd",
-    "hash.js": "^1.1.3",
     "pako": "^1.0.6",
     "seek-bzip": "github:openpgpjs/seek-bzip#6187fc025851d35c4e104a25ea15a10b9b8d6f7d",
+    "sinon": "^4.3.0",
+    "text-encoding-utf-8": "^1.0.2",
     "tweetnacl": "github:openpgpjs/tweetnacl-js#1ef755f2b252a3e328ac739848d00e0dad76be2d",
     "web-stream-tools": "github:openpgpjs/web-stream-tools#dc4b05e8a272b45819233f3df735423432beacfc",
-    "email-addresses": "github:openpgpjs/email-addresses#686743c6452b44bafcd06d47db7f36ddf3f3f118"
+    "whatwg-fetch": "^2.0.3"
```

Comment:
This is pretty nitpicky, but could you keep these packages in the order they were in? I know they're usually in alphabetic order, but the order they were in was to roughly separate what was needed only for development (everything up to `sinon`) and what packages end up in a build (everything starting from `text-encoding-utf-8`).
Also, even if you disagree with that, keeping them in the same order in this PR would make it easier to see what changed here.

Reply:
Npm kinda do it by itself, once I run `npm install elliptic`. May be we could switch to the new ordering convention? It is bothersome to rearrange lines manually after each change, or switching to another git branch.

Reply:
> May be we could switch to the new ordering convention?

Yeah, we could do that, but preferably not in this PR, since it makes it harder to see what changed.
Once we switch to ES6 modules, we'll move the `devDependencies` starting from `text-encoding-utf-8` to `dependencies`, so then is probably a good time to fix the ordering.